JenkinsfileRT

```diff
@@ -78,8 +78,8 @@ bc0.pip_reqs_files = ['requirements-sdp.txt']
 bc0.build_cmds = [
     "pip install certifi -U --force-reinstall",
     "pip install -e .[test,sdp] --no-cache-dir",
-    "pip install git+https://github.com/spacetelescope/stdatamodels.git@master git+https://github.com/spacetelescope/stcal.git@main",
     "pip install pytest-xdist",
+    "pip install -r requirements-sdp.txt",
```

Comment:
Actually, I can try just installing it here; if this works, then we can also use the master commits of the other dependencies.